app/gateway/routing/expressions.md

```diff
@@ -12,49 +12,52 @@ products:
 related_resources:
   - text: Route entity
     url: /gateway/entities/route/
-  - text: Routing in {{site.base_gateway}}
-    url: /gateway/routing/
+  - text: Expressions router reference
+    url: /gateway/routing/expressions-router-reference/
   - text: Expressions repository
     url: https://github.com/Kong/atc-router
 
+min_version:
+  gateway: '3.0'
+
 breadcrumbs:
   - /gateway/
-  - /gateway/routing/
----
-
-{{ page.description | liquify }} Expressions can be used to perform tasks such as defining
-complex routing logic.
-This guide is a reference for the expressions language and explains how it can be used.
 
-## About the expressions language
+faqs:
+  - q: When should I use the expressions router in place of (or alongside) the traditional router?
+    a: We recommend using the expressions router if you are running {{site.base_gateway}} 3.0.x or later. After enabling expressions, traditional match fields on the Route object (such as `paths` and `methods`) remain configurable. You may specify Expressions in the new `expression` field. However, these cannot be configured simultaneously with traditional match fields. Additionally, a new `priority` field, used in conjunction with the expression field, allows you to specify the order of evaluation for Expression Routes.
+  - q: How do I enable the expressions router?
+    a: |
+      In your [kong.conf] <!--TODO link to kong.conf--> file, set `router_flavor = expressions` and restart your {{site.base_gateway}}. Once the router is enabled, you can use the `expression` parameter when you're creating a Route to specify the Routes. For example:
+      ```sh
+      curl --request POST \
+      --url http://localhost:8001/services/example-service/routes \
+      --header 'Content-Type: multipart/form-data' \
+      --form-string name=complex_object \
+      --form-string 'expression=(net.protocol == "http" || net.protocol == "https") &&
+                    (http.method == "GET" || http.method == "POST") &&
+                    (http.host == "example.com" || http.host == "example.test") &&
+                    (http.path ^= "/mock" || http.path ^= "/mocking") &&
+                    http.headers.x_another_header == "example_header" && (http.headers.x_my_header == "example" || http.headers.x_my_header == "example2")'
```

Comment:
This shouldn't be hidden in an FAQ. Also, this is the third time this example is being used again, and it's unnecessary. You've already established that people want to use or test the expressions router; there's no need to show the example yet again.

Reply:
I made a separate section for this at the top of the page and removed the example